recipe/meta.yaml

```diff
@@ -36,6 +38,9 @@ test:
     - test -f "${PREFIX}/lib/libgfortran.dylib"                                   # [osx]
     - test -f "${PREFIX}/lib/libgfortran.{{ libgfortran_version[0] }}.dylib"      # [osx]
 
+    - test -f "${PREFIX}/lib/libgomp.dylib  # [osx]
+    - test -f "${PREFIX}/lib/libgomp.{{ libgcc_s_version[0] }}.dylib  # [osx]
+
```

Comment:
It looks like the tests failed due to missing quotes at the end of the dylib paths.

Reply:
Thanks! I think I got it right in the last commit.